test/extended/cli/setimage.go

```diff
@@ -0,0 +1,131 @@
+package cli
+
+import (
+	"os"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cli] oc set image", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		deploymentConfig = exutil.FixturePath("testdata", "cmd", "test", "cmd", "testdata", "test-deployment-config.yaml")
```

Comment:
Nit: use `exutil.FixturePath("testdata", "test-deployment-config.yaml")` instead, I'd like to get rid of that `testdata/cmd/` directory in the future entirely, so we should use files from `testdata` only.